examples/infrastructure/eks-cluster.yaml

```diff
@@ -42,13 +42,9 @@ Parameters:
     Description: The EC2 Key Pair to allow SSH access to the instances
     Type: AWS::EC2::KeyPair::KeyName
 
-  NodeImageId:
-    Type: AWS::EC2::Image::Id
-    Description: AMI id for the node instances.
-
 Mappings:
 
```

Comment:
Nit: I would comment a link to the following [doc page](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html) for anyone who comes along to update this template

Reply:
Done.